src/components/home.js

```diff
@@ -40,24 +40,32 @@ function Home(props) {
   const getStates = async () => {
     try {
       const [
-        response,
+        {data},
         stateDistrictWiseResponse,
         {data: statesDailyResponse},
         updateLogResponse,
-        stateTestResponse,
+        {data: stateTestData},
       ] = await Promise.all([
         axios.get('https://api.covid19india.org/data.json'),
         axios.get('https://api.covid19india.org/state_district_wise.json'),
         axios.get('https://api.covid19india.org/states_daily.json'),
         axios.get('https://api.covid19india.org/updatelog/log.json'),
         axios.get('https://api.covid19india.org/state_test_data.json'),
       ]);
-      setStates(response.data.statewise);
+      setStates(data.statewise);
       const ts = parseStateTimeseries(statesDailyResponse);
-      ts['TT'] = preprocessTimeseries(response.data.cases_time_series); // TT -> India
+      ts['TT'] = preprocessTimeseries(data.cases_time_series); // TT -> India
       setTimeseries(ts);
-      setLastUpdated(response.data.statewise[0].lastupdatedtime);
-      setStateTestData(stateTestResponse.data.states_tested_data.reverse());
+      setLastUpdated(data.statewise[0].lastupdatedtime);
+      const testData = stateTestData.states_tested_data.reverse();
+      const totalTest = data.tested[data.tested.length - 1];
+      testData.push({
```

Comment:
@KrishnaPravin Are you calculating the total tests numbers or you updated the Api repo to utilize the total numbers?

Reply:
Hi @SensehacK 
I did not calculate. I used `https://api.covid19india.org/data.json` this API to get the testing numbers for India. It was mentioned in the issue -- https://github.com/covid19india/covid19india-react/issues/1013#issuecomment-612651626